src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt

```diff
@@ -0,0 +1,333 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.zoeken
+
+import jakarta.enterprise.inject.Instance
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+import jakarta.transaction.Transactional
+import net.atos.client.zgw.drc.DrcClientService
+import net.atos.client.zgw.drc.model.EnkelvoudigInformatieobjectListParameters
+import net.atos.client.zgw.shared.ZGWApiService
+import net.atos.client.zgw.shared.model.Results
+import net.atos.client.zgw.shared.util.URIUtil
+import net.atos.client.zgw.zrc.ZrcClientService
+import net.atos.client.zgw.zrc.model.Zaak
+import net.atos.client.zgw.zrc.model.ZaakListParameters
+import net.atos.zac.app.task.model.TaakSortering
+import net.atos.zac.flowable.task.FlowableTaskService
+import net.atos.zac.shared.model.SorteerRichting
+import net.atos.zac.util.UriUtil
+import net.atos.zac.zoeken.converter.AbstractZoekObjectConverter
+import net.atos.zac.zoeken.model.ZoekObject
+import net.atos.zac.zoeken.model.index.ZoekObjectType
+import nl.lifely.zac.util.AllOpen
+import org.apache.solr.client.solrj.SolrClient
+import org.apache.solr.client.solrj.SolrQuery
+import org.apache.solr.client.solrj.SolrServerException
+import org.apache.solr.client.solrj.impl.Http2SolrClient
+import org.apache.solr.client.solrj.response.QueryResponse
+import org.apache.solr.common.SolrDocument
+import org.apache.solr.common.params.CursorMarkParams
+import org.eclipse.microprofile.config.ConfigProvider
+import org.flowable.task.api.Task
+import java.io.IOException
+import java.util.UUID
+import java.util.logging.Logger
+
+@Singleton
+@Transactional
```

Comment:
Really transactional? Only needed if we are updating our own database I would think and not for updating a Solr index?